android/docs/guides/installation.md

```diff
@@ -0,0 +1,91 @@
+
+The app evaluates Image Classification via MobileNet V1 (float & quantized),
+using any subset of images used in the
+[ILSVRC 2012 image classification task](http://www.image-net.org/download-images/).
+
+To download the 2012 dataset, you will need to do the following:
+
+*   Open the
+    [ImageNet downloads link](http://www.image-net.org/download-images),
+    creating an ImageNet account if you have not done so already.
+*   Navigate to the 2012 downloads page under the **`Download links to ILSVRC
+    image data`** section.
+*   Download the **`Validation images (all tasks)`** archive.
+*   Create a staging directory for the dataset, call it
+    `${WORKING_DIR}/imagenet`.
+*   From the downloaded archive, extract the images into a
+    `${WORKING_DIR}/imagenet/img` directory.
+
+The full set of validation images is quite large, over 6GB. This can take an
+extremely long time to process, so it is *highly* recommended that you use a
+smaller subset of images. We recommend leaving ~300 images in
+`${WORKING_DIR}/imagenet/img`. The resulting folder contents should appear like:
+
+*   `imagenet/img` : `folder` \
+    Contains the set of images to evaluate over. This could be any subset of the
+    ILSVRC 2012 evaluation dataset (strongly recommend using <300 for test/dev
+    purposes).
+
+The classification model and validation label files are already in the app, for
+convenience. Once the data has been prepared, push the resulting `imagenet`
+folder to `/sdcard/mlperf_datasets/imagenet`.
+
+#### Coco (Detection)
+
+The app evaluates Object Detection using any subset of images used in
+[COCO 2017 dataset](http://cocodataset.org/#download).
+
+To prepare the dataset for object detection you need to do the following:
+
+*   Download the 2017 Val images from above website.
+*   Extract the images to `${WORKING_DIR}/coco/img` directory.
+
+The resulting folder contents should appear like:
+
+*   `coco/img` : `folder` \
+    Contains the set of images to evaluate over. This could be any subset of the
+    COCO 2017 evaluation dataset.
+
+Once the data has been prepared push the resulting `coco` folder to
+`/sdcard/mlperf_datasets/coco`.
+
+## <a name="app-execution"></a> App Execution
+
+After the app has been installed, and the optional dataset(s), simply open
+**`MLPerf Mobile`** from the Android launcher. The app UI allows selection of
+supported tasks, and execution of the supported models (float/quantized) for
+each task. If a dataset has been pushed for the task, the task-specific accuracy
+will be measured, otherwise it will only report latency results.
+
+The user can also configure runtime execution with the following options:
+
+*   TensorFlow Lite Accelerator:
+    *   None - Default (CPU) backend for TensorFlow Lite
+    *   GPU - Enables use of the GPU (via OpenGL/OpennCL)
+    *   NNAPI - Enables use of
+        [Android's NN API](https://developer.android.com/ndk/guides/neuralnetworks)
+*   Number of threads - for CPU execution
```

Comment:
These are only applicable if the user is using a phone that uses the TFLite backend.